ledger-service/http-json/src/failurelib/scala/http/FailureTests.scala

```diff
@@ -46,9 +49,61 @@ abstract class FailureTests
   private def headersWithParties(actAs: List[domain.Party]) =
     Future successful headersWithPartyAuth(actAs, List(), Some(ledgerId().unwrap))
 
-  // TEST_EVIDENCE: Availability: Command submission succeeds after reconnect
-  "Command submission succeeds after reconnect" in withHttpService[Assertion] {
+  val availabilitySecurity: SecurityTest =
+    SecurityTest(property = Availability, asset = "Ledger Service HTTP JSON")
+
+  "Command submission succeeds after reconnect" taggedAs availabilitySecurity in withHttpService[
+    Assertion
+  ] { (uri, encoder, _, client) =>
+    for {
+      p <- allocateParty(client, "Alice")
+      (status, _) <- headersWithParties(List(p)).flatMap(
+        postCreateCommand(
+          accountCreateCommand(p, "23"),
+          encoder,
+          uri,
+          _,
+        )
+      )
+      _ = status shouldBe StatusCodes.OK
+      _ = proxy.disable()
+      (status, output) <- headersWithParties(List(p))
+        .flatMap(postCreateCommand(accountCreateCommand(p, "24"), encoder, uri, _))
+      _ = status shouldBe StatusCodes.ServiceUnavailable
+      (status, out) <- getRequestEncoded(uri.withPath(Uri.Path("/readyz")))
+      _ = status shouldBe StatusCodes.ServiceUnavailable
+      _ = out shouldBe
+        """[-] ledger failed (io.grpc.StatusRuntimeException: UNAVAILABLE: io exception)
+            |[+] database ok
+            |readyz check failed
+            |""".stripMargin.replace("\r\n", "\n")
+      _ <- inside(output) { case JsObject(fields) =>
+        inside(fields.get("status")) { case Some(JsNumber(code)) =>
+          code shouldBe 503
+        }
+      }
+      _ = proxy.enable()
```

Comment:
Given that you are using toxiproxy to inject failures on the connection, I would expect this is more of a ReliabilityTest than a security test.

Reply:
Yes, I think you are right. But since it was tagged as a Availability test before I rather would like to create a separate issue for it.

Reply:
Note, I created a new issue for this: https://github.com/digital-asset/daml/issues/15714